modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyQuery.jsx

```diff
@@ -75,6 +78,12 @@ function VocabularyQuery(props) {
   let infoboxRef = useRef();
   let browserRef = useRef();
 
+  // Update a list of currently selected terms upon any interaction with the multiple choice list
+  useEffect(() => {
+    // Clear input field if maxAnswers=1
+    questionDefinition?.maxAnswers === 1 && inputValue && initialSelection && inputValue != initialSelection[0][1] && setInputValue("");
```

Comment:
This doesn't look right, shouldn't it be `initialSelection[0][0]` (check the label, not the path)?

```suggestion
    questionDefinition?.maxAnswers === 1 && inputValue && initialSelection && inputValue != initialSelection[0][LABEL_POS] && setInputValue("");
```